src/Config.php

```diff
@@ -24,15 +24,16 @@
  */
 class Config
 {
-    public const SETTING_BOOTSTRAP      = 'bootstrap';
-    public const SETTING_COLORS         = 'ansi-colors';
-    public const SETTING_GIT_DIR        = 'git-directory';
-    public const SETTING_INCLUDES       = 'includes';
-    public const SETTING_INCLUDES_LEVEL = 'includes-level';
-    public const SETTING_RUN_EXEC       = 'run-exec';
-    public const SETTING_RUN_MODE       = 'run-mode';
-    public const SETTING_RUN_PATH       = 'run-path';
-    public const SETTING_VERBOSITY      = 'verbosity';
+    public const SETTING_BOOTSTRAP        = 'bootstrap';
+    public const SETTING_COLORS           = 'ansi-colors';
+    public const SETTING_GIT_DIR          = 'git-directory';
+    public const SETTING_INCLUDES         = 'includes';
+    public const SETTING_INCLUDES_LEVEL   = 'includes-level';
+    public const SETTING_RUN_EXEC         = 'run-exec';
+    public const SETTING_RUN_MODE         = 'run-mode';
+    public const SETTING_RUN_PATH         = 'run-path';
+    public const SETTING_VERBOSITY        = 'verbosity';
+    public const SETTING_EXEC_ALL_ACTIONS = 'exec-all-actions';
```

Comment:
I'm not sure about the naming.I think `FAIL_AT_FIRST_ERROR` default `true`  is more intuitive.